scripts/pinned_build.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Mandel Pinned Build"
 
-if [[ $NAME != "Ubuntu" ]]
+if [[ $(uname -v) != *Ubuntu* ]]
```

Comment:
fwiw this may not yield expected behavior when running ubuntu in a container. Also, how safe is it for `uname -v` to not be in quotes here. Locally I see spaces in that
```
$ uname -v
#1 SMP PREEMPT_DYNAMIC Thu, 09 Jun 2022 16:14:10 +0000
```

Reply:
I was expecting something more like:

```
#!/bin/sh

if [ -e /etc/os-release ]
then
        . /etc/os-release
fi

echo $ID
```

Reply:
This will not work.  I am running a Ubuntu lxc container on a debian host.  `uname -a shows` Debian.  It is not debian (only the kernel is Debian).

Reply:
Thanks @matthewdarwin. I have changed to use os-release and provide specific reason when a build is not support.